### DISCUSSION TITLE
Step by step tutorial

### DISCUSSION THREAD

**Discussion by rpermana1976** [Leantime/leantime#692] [General]
is there any step-by-step tutorial installing leantime using xampp on windows?

**Comment by gloriafolaron** [comment]
@rpermana1976 were you able to find a solution for what you needed?

**Reply by abhishekbeesony** [reply]
Hi @rpermana1976 . Did you manage to run the app on Xamp? 

Thank you

**Comment by LeviLucena** [comment]
também busco essa resposta, alguém?
